Packs/Cisco-umbrella-cloud-security/Integrations/CiscoUmbrellaCloudSecurityv2/CiscoUmbrellaCloudSecurityv2_test.py

```diff
@@ -418,3 +418,27 @@ def test_delete_destination_list_command(requests_mock, mock_client):
 
     assert command_results.readable_output == expected_readable_output
     assert command_results.raw_response == response
+
+
+def test_get_access_token(requests_mock, mock_client):
+    """
+    Scenario:
+    - Test get an access token
+    Given:
+    - Nothing
+    When:
+    - get_access_token
+    Then:
+    - Ensure that an access token is returned
+    """
+    
+    response = load_mock_response('token.json')
```

Comment:
no need to add it as a file since it's a small test, define the response as the dict inside the token_json.

Reply:
Okay, I deleted token.json and added the dict within the function